packages/core/src/types/runtime.ts

```diff
@@ -64,6 +64,15 @@ export interface DataAccessor<T = DefaultType> {
   nextBatch: (batchSize: number) => Promise<Array<Sample<T>> | null>;
   seek: (pos: number) => Promise<void>;
 }
+export interface RandomDataAccessor<T = DefaultType> extends DataAccessor {
```

Comment:
`RandomDataAccessor` is a little confusing. It extends the `DataAccessor` and realizes random reading, but it is the real **data accessor** for the scripts, so I suggest rename it to `DataAccessor` and rename origin `DataAccessor` to `SequentialDataAccessor`， @WenheLI @rickycao-qy @yorkie How do you think?